### PR TITLE
Made a working first example.

### PR DESCRIPTION
**Pull request by damiantoczek**
- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

The previous example (with callbacks) did work for me when moving from ExpressJS to Fastify.
I can't tell if another code will have the same problem but mine works.
https://github.com/fastify/help/issues/504

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -35,16 +35,11 @@ fastify.register(require('fastify-mongodb'), {
...
 
-  function onCollection (err, col) {
-    if (err) return reply.send(err)
-
-    col.findOne({ id: req.params.id }, (err, user) => {
-      reply.send(user)
-    })
-  }
+  users.findOne({ id: req.params.id }, (err, user) => {
+    reply.send(user)
```
Comment (Eomm) [inline README.md:41]:
```suggestion
    if (err) {
      reply.send(err)
      return
    }
    reply.send(user)
```

File: README.md [hunk 2]
```diff
@@ -35,16 +35,11 @@ fastify.register(require('fastify-mongodb'), {
 
 fastify.get('/user/:id', function (req, reply) {
   // Or this.mongo.client.db('mydb')
```
Comment (climba03003) [inline README.md:37]:
```suggestion
  // Or this.mongo.client.db('mydb').collection('users')
```

**Review by climba03003 — CHANGES_REQUESTED** [review]

**Comment by damiantoczek** [comment]
Added requested changes

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by climba03003 — APPROVED** [review]
LGTM.

**Pull request merged**